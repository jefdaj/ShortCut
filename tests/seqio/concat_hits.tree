.
|-- cache
|   |-- concat
|   |   |-- emptybht.txt -> ../../cache/lines/...
|   |   `-- fe89e44950.txt -> ../../cache/lines/...
|   |-- each
|   |   `-- e91cdf80df
|   |       |-- 58ea568aa7.bht -> ../../../exprs/psiblast_db/d08318db81_ee63720be0_58ea568aa7.bht
|   |       |-- 58ea568aa7.bht.args -> ../../../cache/lines/...
|   |       |-- d67a110ed4.bht -> ../../../exprs/psiblast_db/d08318db81_ee63720be0_d67a110ed4.bht
|   |       `-- d67a110ed4.bht.args -> ../../../cache/lines/...
|   |-- load
|   |   |-- 82645ed6f6.faa -> $WORKDIR/data/7942gene.faa
|   |   |-- 8aa4a9d2fc.faa -> $WORKDIR/data/6803genes.faa
|   |   `-- a810086108.faa -> $WORKDIR/data/7942genes.faa
|   |-- makeblastdb
|   |   |-- a1512fc008
|   |   |   |-- a1512fc008.pdb.phr
|   |   |   |-- a1512fc008.pdb.pin
|   |   |   `-- a1512fc008.pdb.psq
|   |   `-- e00fb07ddb
|   |       |-- e00fb07ddb.pdb.phr
|   |       |-- e00fb07ddb.pdb.pin
|   |       `-- e00fb07ddb.pdb.psq
|   `-- psiblast
|-- exprs
|   |-- concat_bht
|   |   `-- 87717c62b0.bht
|   |-- list
|   |   |-- 10cb52d92f.faa.list -> ../../cache/lines/...
|   |   |-- be93af8230.pdb.list -> ../../cache/lines/...
|   |   `-- d557a7caa8.faa.list -> ../../cache/lines/...
|   |-- load_faa
|   |   |-- 33c746245c.faa -> ../../cache/load/82645ed6f6.faa
|   |   |-- 387f499ddc.faa -> ../../cache/load/a810086108.faa
|   |   `-- dee24d6c3c.faa -> ../../cache/load/8aa4a9d2fc.faa
|   |-- makeblastdb_prot
|   |   |-- 27a934c58e.pdb -> ../../cache/lines/...
|   |   `-- ced061f0b2.pdb -> ../../cache/lines/...
|   |-- num
|   |   `-- c23938f494.num -> ../../cache/lines/...
|   |-- psiblast_db
|   |   |-- d08318db81_ee63720be0_58ea568aa7.bht
|   |   `-- d08318db81_ee63720be0_d67a110ed4.bht
|   |-- psiblast_db_each
|   |   `-- f51c91829c_4d0e43d616_94686d4973.bht.list -> ../../cache/lines/...
|   `-- str
|       |-- 1d8a61c770.str -> ../../cache/lines/...
|       |-- b9eaf59b0c.str -> ../../cache/lines/...
|       `-- ec3b181d8a.str -> ../../cache/lines/...
`-- vars
    |-- hitlists.bht.list -> ../exprs/psiblast_db_each/f51c91829c_4d0e43d616_94686d4973.bht.list
    |-- pcc6803.pdb -> ../exprs/makeblastdb_prot/27a934c58e.pdb
    |-- pcc7942.pdb -> ../exprs/makeblastdb_prot/ced061f0b2.pdb
    |-- query.faa -> ../exprs/load_faa/33c746245c.faa
    `-- result -> ../exprs/concat_bht/87717c62b0.bht

19 directories, 36 files
