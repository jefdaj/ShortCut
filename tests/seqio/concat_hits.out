Synpcc7942_0002	BAA17503.1	23.497	183	133	3	13	190	10	190	1.89e-07	48.1
Synpcc7942_0002	Synpcc7942_0002	100.000	290	0	0	1	290	1	290	0.0	579
Synpcc7942_0002	Synpcc7942_2044	30.556	72	47	1	91	159	91	162	8.21e-04	36.6

